scripts/build.sh

```diff
@@ -2,85 +2,32 @@
 #
 # This script builds the application from source for multiple platforms.
 # Inspired from hashicorp/terraform build script
+# https://github.com/hashicorp/terraform/blob/83e6703bf77f60660db4465ef50d30c633f800f1/scripts/build.sh
 set -eo pipefail
 
-echo "Bash: ${BASH_VERSION}"
-
 # By default build for dev
 ENV=${ENV:-"dev"}
 
-# Get the git commit
-GIT_COMMIT=$(git rev-parse HEAD)
-GIT_DIRTY=$(test -n "`git status --porcelain`" && echo "-dev" || true)
-
-# Retrieve
-VERSION=$(git describe --exact-match 2>/dev/null || git rev-parse --short HEAD)
-
-# Inject version number
-LD_FLAGS="-X github.com/cloudskiff/driftctl/pkg/version.version=${VERSION}"
-
-# Reference:
-# https://github.com/golang/go/blob/master/src/go/build/syslist.go
-os_archs=(
-    darwin/amd64
-    darwin/arm64
-    linux/386
-    linux/amd64
-    linux/arm
-    linux/arm64
-    windows/386
-    windows/amd64
-)
-
-if [ $ENV != "release" ]; then
-    echo "+ Building env: dev"
-    # If its dev mode, only build for ourself
-    os_archs=("$(go env GOOS)/$(go env GOARCH)")
-    # And set version to git commit
-    VERSION="${GIT_COMMIT}${GIT_DIRTY}"
-fi
-
-if [ -n "$OS_ARCH" ]; then
-  os_archs=("$OS_ARCH")
-fi
-
-echo "ARCH: ${os_archs[*]}"
+# Get the parent directory of where this script is.
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do SOURCE="$(readlink "$SOURCE")"; done
+SELFPATH="$(cd -P "$(dirname "$SOURCE")/.." && pwd)"
 
-# In release mode we don't want debug information in the binary
-# We also set the build env to release
-if [ $ENV = "release" ]; then
-    echo "+ Building env: release"
-    LD_FLAGS="-s -w -X github.com/cloudskiff/driftctl/build.env=release ${LD_FLAGS}"
-fi
+# Change into that directory
+cd "$DIR"
```

Comment:
I don't get why we have to change dir 
![image](https://user-images.githubusercontent.com/6154987/126460640-e6328d5a-c26f-44ea-900d-f1822872b616.png)

Reply:
So goreleaser can find the config file relatively to where the script is located, not from where it's ran

Reply:
Is it really usefull ? I mean the script is meant to be run by the Makefile so the PWD is always the Makefile PWD. I think we can remove theses lines